python_benchmarks/test_huggingface_attn_fwd.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+from nvfuser import FusionDefinition, DataType
+from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
+from .core import run_benchmark, clear_cuda_cache
+import torch
+from .global_params import generate_attn_inputs, FLOAT_DTYPES, PROMOTE_DTYPES
+
+
+# Fusion from huggingface attention implementation
+# https://github.com/Lightning-AI/lightning-thunder/blob/main/thunder/tests/hf_bart_self_attn.py#L73-L83
+def huggingface_attn_fwd_fusion(
```

Comment:
The name suggests it's the full MHA module, but the code seems to be just the non-matmul portion of it? Maybe change the name so readers don't get the wrong impression.

Reply:
Right. The matmul portion does not show up in the nvfuser definitions.